sample.py

```
@@ -0,0 +1,26 @@
+
+class myclass:
+	def __init__(self):self.Data=[1,"two",3.0];self.v=42 
+
+	def run(self):print(self.Data);print(DoSomething(5,7))
+
+def main(): 
+	for i in range(5):print(i*2+3)
+
+main() 
+class myclass:
+	def __init__(self):self.Data=[1,"two",3.0];self.v=42 
```

Ruff (0.14.8)

Multiple statements on one line (semicolon)

(E702)